_typos.toml

```diff
@@ -0,0 +1,14 @@
+[default]
+extend-ignore-identifiers-re = [
```

Comment:
the regex below allow more patterns to be seen as not being typos ?

Reply:
Yes, correct.